ui/framework/src/ui-framework.ts

```diff
@@ -88,6 +88,8 @@ export * from "./ui-framework/hooks/useActiveViewport";
 export * from "./ui-framework/hooks/useAvailableUiItemsProviders";
 export * from "./ui-framework/hooks/useAnalysisAnimationDataProvider";
 export * from "./ui-framework/hooks/useFrameworkVersion";
+export * from "./ui-framework/hooks/useInterval";

```

Comment:
@DanEastBentley @NancyMcCallB - Do you guys think these hooks should be moved to ui-core with the other utility hooks just in case we find a need for them outside of ui-framework?

@mdastous-bentley - can you add unit testing of the new hooks?

Reply:
Yes, if the hook is common enough and is not ui-framework specific, it should be in ui-core/utils/hooks.

Reply:
@DanEastBentley Should I keep the name 'usePopup' ?  Im hesitating to rename it 'useBrowserPopup' so there is no confusion with other popups we have

Reply:
The other popup is not a cross origin popup, it must be from same origin, so maybe yours should be useCrossOriginPopup?